web/index.html

```diff
@@ -1,11 +1,21 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 	<head>
-		<meta charset="UTF-8">
+		<meta charset="UTF-8" />
```

Comment:
I prefer no trailing slashes.

Reply:
Sorry about that. That's just GNOME Builder giving me errors for absolutely no reason.